solr/benchmark/src/test/org/apache/solr/bench/DockMakerTest.java

```diff
@@ -31,7 +30,7 @@
 
   @Before
   public void setup() {
-    System.setProperty("randomSeed", Long.toString(new Random().nextLong()));
+    System.setProperty("randomSeed", Long.toString(random().nextLong()));
```

Comment:
Too bad `com.carrotsearch.randomizedtesting.RandomizedContext#getRunnerSeed` isn't public but any way it's not a big deal; the change here is fine.  `com.carrotsearch.randomizedtesting.RandomizedContext#getRunnerSeedAsString` is public but it's hex.

Reply:
I didn't want to change the code so I just replaced the bad Random ctor.
I don't know what this property means and where it is used, so I left as is.

Reply:
In fact, it can't get worse when switching away from a new Random. 😉